queries.sql

```diff
@@ -93,4 +93,22 @@ SELECT a.name AS digimon_name FROM animals AS a JOIN species AS s ON a.species_i
 
 SELECT a.name AS animal_name FROM animals AS a JOIN owners AS o ON a.owner_id = o.id WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
 
-SELECT o.full_name AS owner_name, COUNT(a.id) AS owned_animals_count FROM owners AS o LEFT JOIN animals AS a ON o.id = a.owner_id GROUP BY o.full_name ORDER BY owned_animals_count DESC LIMIT 1;
\ No newline at end of file
+SELECT o.full_name AS owner_name, COUNT(a.id) AS owned_animals_count FROM owners AS o LEFT JOIN animals AS a ON o.id = a.owner_id GROUP BY o.full_name ORDER BY owned_animals_count DESC LIMIT 1;
+
+SELECT a.name AS animal_name FROM animals a INNER JOIN visits v ON a.id = v.animal_id INNER JOIN vets vet ON v.vet_id = vet.id WHERE vet.name = 'William Tatcher' ORDER BY v.visit_date DESC LIMIT 1;
+
```

Comment:
- [ ] Please add the screenshot of the result of the question **Who was the last animal seen by William Tatcher?** to the PR description.